Core/Base/BaseRSSDB.py

```diff
@@ -0,0 +1,299 @@
+""" BaseRSSDB:
```

Comment:
I think this file should be called something like "SQLAlchemyDB.py". It will be used for the moment by the RSS DBs, but potentially as a base for all the DBs developed using SQLAlchemy.

Reply:
Yes I will resolve it.

Reply:
Renamed `BaseRSSDB.py` to `SQLAlchemyDB.py`